baseapp/baseapp.go

```diff
@@ -356,22 +374,26 @@ func splitPath(requestPath string) (path []string) {
 	return path
 }
 
-// Implements ABCI.
-// Delegates to CommitMultiStore if it implements Queryable
+// Query implements the ABCI interface. It delegates to CommitMultiStore if it
+// implements Queryable.
 func (app *BaseApp) Query(req abci.RequestQuery) (res abci.ResponseQuery) {
```

Comment:
Would be really nice to have the ABCI query functionality documented in detail for each module and query. cc @mossid